nautobot/extras/templates/extras/scheduledjob.html

```diff
@@ -0,0 +1,183 @@
+{% extends 'generic/object_detail.html' %}
+{% load buttons %}
+{% load custom_links %}
+{% load helpers %}
+{% load plugins %}
+
+{% block header %}
+    <div class="row">
+        <div class="col-sm-8 col-md-9">
+            <ol class="breadcrumb">
+                <li><a href="{% url 'extras:scheduled_jobs_list' %}">Scheduled Jobs</a></li>
+                <li>{{ object.name }}</li>
+            </ol>
+        </div>
+        <div class="col-sm-4 col-md-3">
+            <form action="{% url 'extras:scheduled_jobs_list' %}" method="get">
+                <div class="input-group">
+                    <input type="text" name="q" class="form-control" />
+                    <span class="input-group-btn">
+                        <button type="submit" class="btn btn-primary">
+                            <span class="mdi mdi-magnify" aria-hidden="true"></span>
+                        </button>
+                    </span>
+                </div>
+            </form>
+        </div>
+    </div>
+    <div class="pull-right">
+        <div class="pull-right noprint">
+            {% plugin_buttons object %}
+            <!-- TODO clone_button, edit_button? -->
+            {% if perms.extras.delete_scheduledjob %}
+                {% delete_button object key='pk' %}
+            {% endif %}
+        </div>
+    </div>
+    <h1>{% block title %}Scheduled Job: {{ object.name }}{% endblock %}</h1>
+    <div class="pull-right noprint">
+        {% custom_links object %}
+    </div>
+    <ul class="nav nav-tabs">
+        <li role="presentation"{% if not active_tab %} class="active"{% endif %}>
+            <a href="{{ object.get_absolute_url }}">Scheduled Job</a>
+        </li>
+    </ul>
+{% endblock %}
```

Comment:
I would have thought that a lot of this header could be inherited as-is from the generic template?